recipes/libsequence/build.sh

```diff
@@ -2,5 +2,7 @@
 
 CPPFLAGS="-I$PREFIX/include $CPPFLAGS" LDFLAGS="-L$PREFIX/lib $LDFLAGS" ./configure --prefix=$PREFIX
 make
+export LD_LIBRARY_PATH=-L$HOME/$PREFIX/lib
```

Comment:
The syntax is based on following other recipes, which use $PREFIX, not $HOME/prefix.  Hopefully no one is installing runtime libs in ~/ :).  I've pushed new changes moving this stuff into the build: field in meta.yaml and upped the build no. to 1.